PathOfBuildingCommunity/PathOfBuilding
# Add max calculcated fuses for Expl Arrow to calcs

### Description of the problem being solved: Expanding upon the ability for PoB to automatically calculate the max number of sustainable fuses for Explosive Arrow that was in #4918 to add the number of sustainable fuses to the calcs tab.  When leveling through an EA build, I was curious as to how many fuses I could sustain but only saw the overall damage go up but had to calculate by hand the number of fuses.  

### Steps taken to verify a working solution:
- Import build with EA skill.  Toggle between Max Sustainable Fuses and manually setting fuses
- Confirm the value does not show when you don't have Max Sustainable Fuses selected for the skill part on the calcs tab

### Link to a build that showcases this PR:
https://pobb.in/ESk0PXDoIwGu

### Before screenshot:
![image](https://user-images.githubusercontent.com/34242968/206023524-d512a3bf-52d2-41bd-9b89-5ba39273225f.png)

### After screenshot:
![image](https://user-images.githubusercontent.com/34242968/206023646-6c2598ad-9e93-4ef2-8af8-066882fd07d6.png)
